.github/workflows/pr-ci-triggers.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  pull_request:
+
+jobs:
+  test:
```

Comment:
job name : `vendor-ci` ?

Reply:
switched it to `vendors-ci-triggers-list`, since i think this would be more descriptive, WDYT?